src/contexts/AuthProvider.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { createContext } from 'react';
+
+import useLocalStorageAuth from 'hooks/useLocalStorageAuth';
+import { Auths } from 'types/auth';
+
+type Nullable<T> = T | null;
+
+interface AuthContextType {
+  auth: Nullable<Auths>;
+  setAuth: (authData: Nullable<Auths>) => void;
+}
+
+const AuthContext = createContext<AuthContextType>({
+  auth: null,
+  setAuth: (_authData: Nullable<Auths>) => {
+    // this is just to prevent lint error
```

Comment:
What lint error might this be preventing? 👀 

We can explicitly ignore specific linting errors using `eslint-disable-next-line` etc.